### ISSUE TITLE
508-defect-4 [COGNITION]: Consider rewriting instructions on the Choose a clinic screen for clarity

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24171]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Peggy, Trevor_

## Details

The Choose a clinic screen repeats the phrase choose a clinic a number of times. I'd like to consider moving the address of the VAMC to the previous screen and rewriting the instructions on this screen to be shorter and emphasize the call to action. Screenshots attached below.

## Acceptance Criteria

- [ ] Content is rewritten and approved by Peggy ~OR~
  - Content on the clinic selection page matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/health-care/covid-vaccine-distro/vaos-for-distribution-initiative/copy-docs/clinic-selection.md) _(5/10-Lauren added)_
- [ ] ~Team decides not to update content and closes ticket with `wontfix` tag~

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/clinic
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/facility

## Screenshots or Trace Logs

![Screen Shot 2021-05-04 at 9 03 40 AM](https://user-images.githubusercontent.com/934879/117023792-09ea1980-acbf-11eb-9c3e-fe42a8f28c22.png)

---
![Screen Shot 2021-05-04 at 9 04 54 AM](https://user-images.githubusercontent.com/934879/117023822-11112780-acbf-11eb-8f72-25a13d6ab199.png)

**Comment by peggygannon** [comment]
@laurenernest @outerpress Let me know if you want to discuss at our next sync? or drop in a recommendation here.

**Comment by outerpress** [comment]
@peggygannon here's good, then we can review if there are any follow up questions at the next sync.

I generally agree with Trevor's feedback on the clinic page. I think we were trying to align this with the [current direct scheduling clinic page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/design/direct-schedule-flow/clinic_page.png), but the address doesn't add anything since they don't need to have been seen at one of the clinics previously.

For the facility selection page, I haven't heard feedback that we need more information to distinguish facilities than city and state. I don't want to add to the cognitive load on that page, so let's leave it as is.

**Comment by peggygannon** [comment]
@outerpress 

General notes: 

1. I noticed in reviewing this screen against the Direct Scheduling screen that the H1s in VAOS Distribution use "a" vs "your"  (Choose a clinic vs. Choose your clinic)
Was there a decision to use "a" vs "your" ? If so, then I recommend all H1s across the different VAOS workflows are reviewed to ensure consistency and alignment. 
Also, the H1s in DS workflow include "VA" We should be consistent across the different VAOS paths. 

2. The H1 on the DS page is = Choose your VA clinic for your {type of appt}
Can we add "VA" for consistency with other VAOS paths? 

---

Here is my recommendation to address accessibility feedback on the clinic screen: 

H1: **Choose a VA clinic**  

Clinics at {name of VAMC} offer vaccine appointments at different times. 

Choose a clinic for your vaccination (*Required)
{radio buttons} name of clinic 1
{radio button} name of clinic 2

cc @1Copenut

**Comment by 1Copenut** [comment]
Thank you @peggygannon. This will definitely smooth out the accessibility cognition issue.

**Comment by outerpress** [comment]
> the H1s in VAOS Distribution use "a" vs "your"  - Was there a decision to use "a" vs "your" ?

I wasn't around during the direct scheduling days - @rtwell do you have any background on this? My instinct is to go with "a" across all flows since the clinics are specific to the facility, not something the user has control over.

> the H1s in DS workflow include "VA" We should be consistent across the different VAOS paths.

Capturing our Slack conversation here, we decided it made more sense to drop "VA" from all flows since it doesn't provide add'l info to help the user make the decision or understand the choice

cc @laurenernest a couple more areas we could align language across flows ^

--------

Updated clinic page with Peggy's recs, @1Copenut @peggygannon :

![HP-chooseclinic-empty.png](https://images.zenhubusercontent.com/5f7ce3163782a585b910eaf3/932c63a4-6d1a-47ae-b004-21e9a69449a4)

**Comment by rtwell** [comment]
oh man @outerpress i don't remember. i would defer to @peggygannon's expertise tho

**Comment by peggygannon** [comment]
Hello! @outerpress @rtwell I recall deciding to use "your" in the DS and Request flows to present a more conversational and personalized tone throughout the flow. 

I know "your" night not work for some H1s (as the user doesn't always have control over the subject). Also "a" is 3 characters less than "your",  so better for mobile:)

My recommendation it to make it consistent across the flows--and we should probably revisit the other H1s to ensure consistency.

**Comment by laurenernest** [comment]
Moving to dev to update the vaccine flow. Will make separate tickets for the other flows.

**Comment by LeKeve** [comment]
@1Copenut This is ready for review.
